site/content/docs/4.3/layout/grid.md

```diff
@@ -365,6 +364,160 @@ You can also use the accompanying Sass mixin, `row-cols()`:
 }
 {{< /highlight >}}
 
+## Gutters
+
+Gutters can be responsively adjusted by breakpoint-specific gutter classes in as well horizontal, vertical and both directions. By default, `.row`s have a horizontal gutter of `1.5rem`. Removing this default gutter is possible by adding the `.g-0` class.
+
+### Changing the gutters
+
+Classes are built from the `$gutters` Sass map which is inherited from the `$spacers` Sass map. Each value in the Sass map generates classes for gutters in vertical, horizontal or both ways.
+
+```sass
+$grid-gutter-width: 1.5rem;
+$gutters: (
+  0: 0,
+  1: $spacer * .25,
+  2: $spacer * .5,
+  3: $spacer,
+  4: $spacer * 1.5,
+  5: $spacer * 3,
+);
+```
+
+### Horizontal gutters
+
+`.gx-*` classes can be used to control the horizontal gutter widths. The `.container` or `.container-fluid` parent may need to be adjusted if larger gutters are used too to avoid unwanted overflow, using a matching padding utility. For example, in the following example we've increased the padding with `.px-lg-4` at the large (`lg`) breakpoint and above:
+
+{{< example >}}
+<div class="container px-lg-4">
+  <div class="row gx-lg-5">
```

Comment:
Not limited to this line, some examples in the Gutters section use `lg` breakpoints, making it difficult to understand on mobile devices. Is it possible to rewrite for mobile friendly examples?

Reply:
Good point, removed the breakpoint specific gutters